### PR TITLE
feat(docs): icons cleanup in docs examples

### PR DESCRIPTION
**Pull request by mnajdova**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

This PR:
- removes examples for `Icon` component
- replaces usage of `Icon` component with icons from `@fluentui/react-northstar-icons` package

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/fluentui/pull/12551)

### CODE REVIEW

File: packages/fluentui/docs/src/examples/components/SvgIcon/States/SvgIconExampleDisabled.shorthand.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import { CallVideoIcon } from '@fluentui/react-icons-northstar';
+
+const IconExampleDisabled = () => (
+  <div>
```
Comment (layershifter) [inline packages/fluentui/docs/src/examples/components/SvgIcon/States/SvgIconExampleDisabled.shorthand.tsx:5]: Is `div` necessary there?

File: packages/fluentui/docs/src/examples/components/SvgIcon/Usage/index.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import ComponentExample from '../../../../components/ComponentDoc/ComponentExample';
+import ExampleSection from '../../../../components/ComponentDoc/ExampleSection';
+
+const Usage = () => (
+  <ExampleSection title="Usage">
+    <ComponentExample
+      title="Theme Icons"
+      description="You can access all icon names available in the current theme."
```
Comment (layershifter) [inline packages/fluentui/docs/src/examples/components/SvgIcon/Usage/index.tsx:9]: Not sure that this valid anymore

File: packages/fluentui/docs/src/examples/components/SvgIcon/Variations/SvgIconExampleBordered.shorthand.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    <SkypeLogoIcon bordered />
+    <EmojiIcon bordered />
+    <PlayIcon bordered />
+    <CallIcon bordered />
+    <CallVideoIcon bordered />
+    <TranslationIcon bordered />
+  </Flex>
+);
+
+export default IconExampleBordered;
```
Comment (layershifter) [inline packages/fluentui/docs/src/examples/components/SvgIcon/Variations/SvgIconExampleBordered.shorthand.tsx:23]: Should be all exports in examples renamed to `SvgIcon*`?

File: packages/fluentui/docs/src/examples/components/SvgIcon/Usage/SvgIconSetExample.shorthand.tsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+};
+
+const SvgIconSetExampleShorthand: React.FunctionComponent<{ themeName: string }> = ({ themeName }) => (
+  <>
+    <div>
+      <Divider>
+        <Header as="h3" content="Regular" />
+      </Divider>
+      <Grid columns={4} styles={{ textAlign: 'center' }}>
+        {Object.values(icons)
```
Comment (layershifter) [inline packages/fluentui/docs/src/examples/components/SvgIcon/Usage/SvgIconSetExample.shorthand.tsx:16]: Will not work in IE 😭

File: packages/fluentui/docs/src/examples/components/SvgIcon/Variations/SvgIconExampleBordered.shorthand.tsx (hunk 3 above)
Comment (mnajdova) [reply packages/fluentui/docs/src/examples/components/SvgIcon/Variations/SvgIconExampleBordered.shorthand.tsx:23]: BTW these examples are not going to be added on the docs for now, will be with next PR (due to folder changes required for the Icon component..)

File: packages/fluentui/docs/src/examples/components/SvgIcon/Usage/SvgIconSetExample.shorthand.tsx [hunk 5]
```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { Grid, Divider, Header, SvgIconProps } from '@fluentui/react-northstar';
+import * as icons from '@fluentui/react-icons-northstar';
+
+const cellStyles = {
+  margin: '10px 0',
+};
+
+const SvgIconSetExampleShorthand: React.FunctionComponent<{ themeName: string }> = ({ themeName }) => (
```
Comment (layershifter) [inline packages/fluentui/docs/src/examples/components/SvgIcon/Usage/SvgIconSetExample.shorthand.tsx:9]:
```suggestion
const SvgIconSetExampleShorthand: React.FunctionComponent = () => (
```

File: packages/fluentui/docs/src/examples/components/SvgIcon/Usage/SvgIconSetExample.shorthand.tsx (hunk 4 above)
Comment (mnajdova) [reply packages/fluentui/docs/src/examples/components/SvgIcon/Usage/SvgIconSetExample.shorthand.tsx:16]: OMG, let's see what lodash has...

File: packages/fluentui/docs/src/examples/components/SvgIcon/Usage/SvgIconSetExample.shorthand.tsx [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+            );
+          })}
+      </Grid>
+    </div>
+  </>
+);
+
+SvgIconSetExampleShorthand.defaultProps = {
+  themeName: 'teams',
+};
```
Comment (layershifter) [inline packages/fluentui/docs/src/examples/components/SvgIcon/Usage/SvgIconSetExample.shorthand.tsx:55]:
```suggestion
```

File: packages/fluentui/docs/src/examples/components/SvgIcon/Usage/SvgIconSetExample.shorthand.tsx (hunk 4 above)
Comment (layershifter) [reply packages/fluentui/docs/src/examples/components/SvgIcon/Usage/SvgIconSetExample.shorthand.tsx:16]: I tried to simplify this example:

```tsx
import { Grid, Divider, Header, SvgIconProps } from '@fluentui/react-northstar';
import * as exports from '@fluentui/react-icons-northstar';
import * as React from 'react';
import * as ReactIs from 'react-is';

const icons = Object.keys(exports).reduce((acc: React.FC<SvgIconProps>[], exportName) => {
  if (ReactIs.isValidElementType(exports[exportName])) {
    acc.push(exports[exportName]);
  }

  return acc;
}, []);

const cellStyles = {
  margin: '10px 0',
};

const SvgIconSetExampleShorthand = () => (
  <>
    <div>
      <Divider>
        <Header as="h3" content="Regular" />
      </Divider>
      <Grid columns={4} styles={{ textAlign: 'center' }}>
        {icons.map(Icon => (
          <div key={Icon.displayName} style={cellStyles}>
            <Icon />
            <br />
            <code>{Icon.displayName}</code>
          </div>
        ))}
      </Grid>
    </div>

    <div>
      <Divider>
        <Header as="h3" content="Outline" />
      </Divider>
      <Grid columns={4} styles={{ textAlign: 'center' }}>
        {icons.map(Icon => (
          <div key={`${Icon.displayName} - outline`} style={cellStyles}>
            <Icon outline />
            <br />
            <code>{Icon.displayName}</code>
          </div>
        ))}
      </Grid>
    </div>
  </>
);

export default SvgIconSetExampleShorthand;
```

**Comment by layershifter** [comment]
Should we also remove all `Icon*` examples in this PR?

File: packages/fluentui/docs/src/examples/components/SvgIcon/Usage/index.tsx (hunk 2 above)
Comment (mnajdova) [reply packages/fluentui/docs/src/examples/components/SvgIcon/Usage/index.tsx]: Changed

File: packages/fluentui/docs/src/examples/components/SvgIcon/States/SvgIconExampleDisabled.shorthand.tsx (hunk 1 above)
Comment (mnajdova) [reply packages/fluentui/docs/src/examples/components/SvgIcon/States/SvgIconExampleDisabled.shorthand.tsx]: Removed

File: packages/fluentui/docs/src/examples/components/SvgIcon/Variations/SvgIconExampleBordered.shorthand.tsx (hunk 3 above)
Comment (mnajdova) [reply packages/fluentui/docs/src/examples/components/SvgIcon/Variations/SvgIconExampleBordered.shorthand.tsx]: Done

**Pull request merged**